go/sqltypes/value.go

```diff
@@ -250,10 +255,12 @@ func (v Value) ToBytes() ([]byte, error) {
 	switch v.typ {
 	case Expression:
 		return nil, vterrors.New(vtrpcpb.Code_INVALID_ARGUMENT, "expression cannot be converted to bytes")
-	case HexVal:
+	case HexVal: // TODO: all the decode below have problem when decoding odd number of bytes. This needs to be fixed.
```

Comment:
What is the issue are you talking about here?

Reply:
So, do odd numbered values don't work right now?

Reply:
If we execute a `ToBytes()` function on them in the vtgate code.
which most likely happen in vindexes code.
So, if a vindex is presented as hex or bit it will not work today for odd string presentation

eg. `0x0f` will fail whereas `0x0f0f` will pass etc.

We should fix this as separate PR.